src/docs/development/tools/vs-code.md

```diff
@@ -145,6 +145,32 @@ running or debugging.
       To customize, click the cog at the top of the
       **Debug Sidebar** to create a `launch.json` file.
       You can then modify the values.
+      
+      
+### Run app in debug, profile, or release mode
+
+Flutter offers many different build modes to run your app in. 
+You can read more about them in [Flutter's build modes][].
+
+ 1. Open the `launch.json` file in VS Code.
+     * If you do not have a `launch.json` file, go to 
+     the Run view in VS Code and click **create a launch.json file**.
```

Comment:
=> Bold the "Run" text. (Bold menu items:  https://developers.google.com/style/ui-elements)